### PR TITLE
Add oserver pattern to branch1

### PR DESCRIPTION
**Pull request by OresytO**

### CODE REVIEW

File: observer/src/ObserverRunner.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import observer.impl.ObserverB;
+import subject.impl.SubjectImpl;
+
+/**
+ * @author OrestO
+ * @since 3/26/2015
+ */
+public class ObserverRunner
+{
+  public static void main(String[] args)
```
Comment (Romanow88) [inline observer/src/ObserverRunner.java:11]: where is java-doc

Comment (OresytO) [inline observer/src/ObserverRunner.java:11]: because it is main

File: observer/src/observer/Observer.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package observer;
+
+/**
+ * @author OrestO
+ * @since 3/26/2015
+ */
```
Comment (Romanow88) [inline observer/src/observer/Observer.java:6]: No javadoc, interface with no methods?

**Comment by Romanow88** [comment]
@OresytO Please provide javadocs to all files prior to pull

**Pull request closed without merging**